src/components/Main.jsx

```diff
@@ -15,6 +16,7 @@ const Main = () => (
       <Route path="/recent" component={RecentSnippets} />
       <Route path="/about" component={About} />
       <Route path="/sign-in" component={SignIn} />
+      <Route path="/snippet/:id" component={Snippet} />
```

Comment:
I think we better use `/:id` to keep backward compatibility with already created snippets (we use URI like http://xsnippet.org/2).

Reply:
removed redundant part of path